README.md

```diff
@@ -51,6 +51,17 @@ $ npx is-my-node-vulnerable
 
 ```
 
+### Output - when unsupported
+
+```console
+$ node -v
+v17.9.1
+$ npx is-my-node-vulnerable
+Not an actively supported Node.js version: v17.9.1
```

Comment:
Could you rephrase it to: "Node.js v17.9.1 is end-of-life. There are high chances of being vulnerable. Please upgrade it."

Reply:
But it might not be true. It could be a non LTS newer version. Like a v19 or v20 that's not active. Right ?

Reply:
I could see if your node is < the lowest current active. And then say too old. Otherwise just say not active support.

Reply:
v19 is completely supported. 

> I could see if your node is < the lowest current active. And then say too old. Otherwise just say not active support.

That won't work. For instance, v15  is end-of-life and v14 is LTS, so the check will be tricky. I think the best way is to change the message as suggested. People won't be able to call `npx is-my-node-vulnerable` with Node.js 20 for instance since it's not live.

Reply:
maybe we need a differnet check if we really want to specify EOL.